ambry-server/src/main/java/com/github/ambry/server/AmbryMain.java

```diff
@@ -45,8 +45,8 @@ public static void main(String[] args) {
           Utils.getObj(clusterMapConfig.clusterMapClusterAgentsFactory, clusterMapConfig,
               options.hardwareLayoutFilePath, options.partitionLayoutFilePath);
       logger.info("Bootstrapping AmbryServer");
-      VcrClusterAgentsFactory vcrClusterAgentsFactory = Utils.getObj(cloudConfig.vcrClusterAgentsFactoryClass);
-      ambryServer = new AmbryServer(verifiableProperties, clusterAgentsFactory, vcrClusterAgentsFactory,
+//      VcrClusterAgentsFactory vcrClusterAgentsFactory = Utils.getObj(cloudConfig.vcrClusterAgentsFactoryClass);
+      ambryServer = new AmbryServer(verifiableProperties, clusterAgentsFactory, (VcrClusterAgentsFactory) null,
```

Comment:
null? Looks like some debug code .

Reply:
Removed it though my local server would not start without this change. How does your server start?

Reply:
I recently relies on integration test to have vcrClusterAgentsFactory created. But `null` also work for local test.